### DISCUSSION TITLE
NegativeHabitType

### DISCUSSION THREAD

**Discussion by ganesshkumar** [iSoron/uhabits#1062] [Feature Requests] [answered]
I saw the recent pull request with HabitType and NumericHabitType got merged.

**Can we have a NegativeHabitType as well?** 

In the NegativeHabitType, the streak and habit score will continue to build up if we don't make an entry and the streak will break if we make an entry. 

I want it primarily to avoid negative habits. Currently I create a task like "Do not do X" and it works. Just checking if we can have this since the NumbericalHabitType got adopted.

**Comment by hiqua** [comment, marked as answer]
Duplicate: https://github.com/iSoron/uhabits/discussions/696
